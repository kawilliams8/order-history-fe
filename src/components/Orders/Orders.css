.Orders {
  width: 80%;
}

.deleteButton {
  background-color: red;
  color: white;
  height: 60px;
  margin-top: 60px;
}