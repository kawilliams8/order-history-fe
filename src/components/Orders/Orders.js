import React from 'react';
import './Orders.css';
import Order from '../Order/Order';

const Orders = ({ data }) => {
  const orderCards = data.map(order => {
    return <Order data={order}/>
  })

  return (
    <main className="Orders">
      {orderCards}
    </main>
  )
}

export default Orders;