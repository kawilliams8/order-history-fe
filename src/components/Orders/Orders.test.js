import React from 'react';
import { shallow } from 'enzyme';
import Orders from '../Orders/Orders';

describe('Orders', () => {
  
  it('should match the snapshot', () => {
    const mockOrders = [
      {id: 1,
      img: "image.png",
      name: "Mock Order 1",
      description: "This is a mock description",
      price: 3
      },
      {id: 2,
      img: 'image.png',
      name: "Mock Order 2",
      description: "This is a mock description",
      price: 45
      }
  ]
    const wrapper = shallow(<Orders data={mockOrders}/>)
    expect(wrapper).toMatchSnapshot();
  })
})