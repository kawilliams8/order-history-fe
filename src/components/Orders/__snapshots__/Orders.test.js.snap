// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Orders should match the snapshot 1`] = `
<main
  className="Orders"
>
  <Order
    data={
      Object {
        "description": "This is a mock description",
        "id": 1,
        "img": "image.png",
        "name": "Mock Order 1",
        "price": 3,
      }
    }
    key="0"
  />
  <Order
    data={
      Object {
        "description": "This is a mock description",
        "id": 2,
        "img": "image.png",
        "name": "Mock Order 2",
        "price": 45,
      }
    }
    key="1"
  />
</main>
`;
